client/hbs/head.hbs

```diff
@@ -77,12 +85,13 @@
 
 
 	{{#if admin}}
-	<!-- EDITOR -->
-	<link rel="stylesheet" href="/css/admin.css">
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.2/ace.js"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
-	<script src="/js/admin.js"></script>
+		<!-- EDITOR -->
+		<link rel="stylesheet" href="/css/admin.css">
```

Comment:
lol nm, that appears to be a bug in github's split diff mode. In the unified version it shows up correctly